app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/EoriLanguageController.scala

```diff
@@ -28,6 +28,8 @@ class EoriLanguageController @Inject() (config: Configuration, languageUtils: La
 
   override protected def fallbackURL: String =
```

Comment:
That fallback url could do with changing to subscribe

Reply:
I'll update it

Reply:
Done